client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -53,7 +53,13 @@ class DeployedModel(object):
     Examples
     --------
     .. code-block:: python
+       :emphasize-lines: 3,4
```

Comment:
fyi line `4` is empty (line `1` would be your `# Preferred method . . .` comment)